package.json

```diff
@@ -39,7 +39,8 @@
     "test:watch": "jest --watchAll --coverage",
     "lint": "tslint -c tslint.json -p tsconfig.json \"src/**/*.ts\"",
     "prettier": "prettier --write \"{src,templates,test}/**/*.{ts,js}\"",
-    "prettier:check": "prettier --check \"{src,templates,test}/**/*.{ts,js}\""
+    "prettier:check": "prettier --check \"{src,templates,test}/**/*.{ts,js}\"",
```

Comment:
Does the `**` actually work correctly here? I had understood that, in bash at least, that's an extension feature that's disabled by default: https://stackoverflow.com/questions/28176590/what-do-double-asterisk-wildcards-mean/28199633#28199633. `fish` and `zsh` support it, but POSIX compatible shells don't use it at all; it's decidedly not portable.

EDIT: It looks like prettier is doing the parsing, in which case it makes perfect sense.